web/src/views/repo/RepoWrapper.vue

```diff
@@ -132,7 +132,7 @@ watch([repoOwner, repoName], () => {
   loadRepo();
 });
 
-const badgeUrl = computed(() => `/api/badges/${repo.value.owner}/${repo.value.name}/status.svg`);
+const badgeUrl = computed(() => `/api/badges/${repo.value?.owner}/${repo.value?.name}/status.svg`);
```

Comment:
When `repo` is undefined the `badgeUrl` will be invalid. I would suggest hiding it in that case.

Reply:
done